pkg/haproxy/config.go

```diff
@@ -85,15 +85,29 @@ frontend stats
   stats refresh 500ms
   stats hide-version
   stats show-legends
-{{ $port := .Port | printf "%d" }}
+{{range .ExtraServices}}
+frontend svc_{{.Name}}
+  mode tcp
+  bind *:{{.Port | printf "%d"}} name lb
```

Comment:
Should the name match frontend?

Reply:
I dont think it needs to be match. But for consistency sake keeping it same